### DISCUSSION TITLE
How to get value from DatePicker like this DD/MM/YYYY ?

### DISCUSSION THREAD

**Discussion by Sakkhor909** [mantinedev/mantine#945] [Q&A] [answered]
Hello, First of all, thank you contributors for this amazing library. I am using DatePicker and got value from there like this `2022-03-07T18:00:00.000Z` But I wanted it like this DD/MM/YYYY only. I know can modify the string by string methods but I want to if there is a proper way for Mantine to do this?
```
   <DatePicker
            placeholder="Pick date"
            label="Release Date"
            {...form.getInputProps("release_date")}
            required
            dropdownType="modal"
            icon={<MdToday />}
            inputFormat="DD/MM/YYYY"
            labelFormat="DD/MM/YYYY"
          />
```

**Comment by rtivital** [comment, marked as answer]
You are converting date to string somewhere in your code. Form values contain Date object – https://codesandbox.io/s/hardcore-dewdney-uz6lxg?file=/src/App.tsx

**Reply by Sakkhor909** [reply]
I am using Fetch and sending this data as a POST method request to save in the database. It looks like that is converting the Date. How can I convert it back to its original form?

**Reply by rtivital** [reply]
You can convert any string to date `new Date(someString)`
